var/spack/repos/builtin/packages/r-phantompeakqualtools/package.py

```diff
@@ -13,18 +13,16 @@ class RPhantompeakqualtools(RPackage):
        package."""
 
     homepage = "https://github.com/kundajelab/phantompeakqualtools"
-    url      = "https://github.com/kundajelab/phantompeakqualtools/raw/master/spp_1.14.tar.gz"
-
-    version('1.14', sha256='d03be6163e82aed72298e54a92c181570f9975a395f57a69b21ac02b1001520b')
+    url      = "https://github.com/hms-dbmi/spp/archive/refs/tags/1.15.2.tar.gz"
```

Comment:
These changes are not right. According to the documentation:
>spp_1.14.tar.gz : modified SPP peak-caller package (The original SPP-peak caller package was written by Peter Kharchenko [2], https://github.com/hms-dbmi/spp)

There is an updated version of `r-phantompeakqualtools` but it still includes `spp_1.14`. If a new version of spp is desired then an `r-spp` package should be created and this package should depend on it, only installing the `run_spp.R` script. Otherwise, this package will use the provided modified spp_1.14 distribution. In either case, if this package is updated it needs to be updated from the https://github.com/kundajelab/phantompeakqualtools URL, where the current version is 1.22.

Also, the old version should not be eliminated from the recipe.